src/definition/users/IUserUpdateContex.ts

```diff
@@ -0,0 +1,7 @@
+import { IUser } from './IUser';
+
+export interface IUserUpdateContext {
+    user: IUser;
+    performedBy?: IUser;
+    previousData?: IUser;
+}
```

Comment:
It would be good to add some comments here explaining app devs what is the purpose of the interface and the properties